heft-plugins/heft-dev-cert-plugin/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "@rushstack/heft-dev-cert-plugin",
+  "version": "0.1.0",
```

```suggestion
  "version": "0.0.0",
```

The first publish will bump this to `0.1.0`